docs/tutorials/new-feature-tutorial.md

```diff
@@ -585,10 +598,25 @@ function dispatch_normal_event(event) {
       };
 ```
 
+For organization settings you shouldn't worry about syncing/updating of
+the input elements to other browser windows as they are automatically
+updated by `settings_org.sync_realm_settings`.
```

Comment:
I think we should probably encourage them to test, still, but I think we can say it should work automatically.